gateways: Add tutorial for failover and failback states (#719)
opnsense/docs: source/manual/how-tos/multiwan.rst

To combine Load Balancing with Failover you will have 2 or more WAN connections
for Balancing purposes and 1 or more for Failover. OPNsense offers 5 tiers
(Failover groups) each tier can hold multiple ISPs/WAN gateways.

----------------------------
Failover and Failback States
----------------------------

> In some Multiwan setups it might be necessary to control firewall states more aggressively. The most common example

Reviewer: ```suggestion
In some multi-WAN setups it may be necessary to directly control firewall states. The most common example
```
